src/Avalonia.Build.Tasks/CompileAvaloniaXamlTask.cs

```diff
@@ -51,8 +51,11 @@ public bool Execute()
 
         private static void CopyAndTouch(string source, string destination)
         {
-            File.Copy(source, destination, overwrite: true);
-            File.SetLastWriteTimeUtc(destination, DateTime.UtcNow);
+            if (File.Exists(source))
```

Comment:
This check should be specific to the PDB copy operation. Failure on the other two operations can only mean an internal error which should be reported, not ignored.

Reply:
I've added a flag, please recheck
@maxkatz6 @TomEdwardsEnscape

Reply:
Works for me.